db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -715,58 +697,92 @@ Status DBImpl::CompactRange(const CompactRangeOptions& options,
     }
     s = RunManualCompaction(cfd, ColumnFamilyData::kCompactAllLevels,
                             final_output_level, options, begin, end, exclusive,
-                            false, max_file_num_to_ignore);
+                            false, port::kMaxUint64);
```

Comment:
Is there a reason why first_overlapped_level and max_overlapped_level shouldn't be applied to universal compaction too? I'm a little bit concerned that we move away from max_file_num_to_ignore here.

Reply:
my understanding is that universal compaction always compacts all levels.

`max_file_num_to_ignore` is always `kMaxUint64` in the original code path.

Reply:
OK.